backend/dataall/modules/datasets/api/__init__.py

```diff
@@ -1,6 +1,7 @@
 """The GraphQL schema of datasets and related functionality"""
 from dataall.modules.datasets.api import (
-    table_column
```

Comment:
Why did we change the naming convention? api graphql Objects were capitalized before right?

Reply:
That's correct and if you think it should remain the existing way please let me know.
I thought that this modularization is a good opportunity to start following the python convention ( underscore instead of capitalized). 
But as I said if you feel that it's not correct (or capitalized is a unique style of data.all  :)) I will return it back